SemanticCompoundQueries.php

```diff
@@ -17,7 +17,7 @@
 
 if ( !defined( 'MEDIAWIKI' ) ) die();
 
-define( 'SCQ_VERSION', '0.4.1' );
+define( 'SCQ_VERSION', '1.0' );
```

Comment:
I think we have been consistent with the versioning for most of the respo's (== 1.0.0).

Aside from this, after a quick (really quick ~ 30 sec.) look at the code I found that `SCQ_QueryResult.php` with its method `addResult` is not being used [0] which suggests dead code (or better a dead class).

So, given the state and quality of the code I'd go with a 0.5 but if you feel comfortable with the quality then go ahead.

[0] https://github.com/SemanticMediaWiki/SemanticCompoundQueries/search?utf8=%E2%9C%93&q=addResult